### ISSUE TITLE
[BUG] Phoenix launch fails on dataframes with boolean columns

### ISSUE THREAD

**Issue by axiomofjoy** [Arize-ai/phoenix#594]
**Describe the bug**
Phoenix fails to launch on dataframes with boolean columns. Users will expect to be able to use booleans, e.g., for binary classification problems.

```
numpy boolean subtract, the `-` operator, is not supported, use the bitwise_xor, the `^` operator, or the logical_xor function instead.

GraphQL request:44:11
43 |           max: dataQualityMetric(metric: max, timeRange: {start: $startTime, end: $endTime})
44 |           psi: driftMetric(metric: psi, timeRange: {start: $startTime, end: $endTime})
   |           ^
45 |         }
Traceback (most recent call last):
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/graphql/execution/execute.py", line 521, in execute_field
    result = resolve_fn(source, info, **args)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/schema/schema_converter.py", line 519, in _resolver
    return _get_result(_source, strawberry_info, **kwargs)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/schema/schema_converter.py", line 511, in _get_result
    return field.get_result(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/field.py", line 169, in get_result
    return self.base_resolver(*args, **kwargs)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/types/fields/resolver.py", line 186, in __call__
    return self.wrapped_func(*args, **kwargs)
  File "/Users/xandersong/phoenix/src/phoenix/server/api/types/Dimension.py", line 52, in drift_metric
    data := get_drift_timeseries_data(
  File "/Users/xandersong/phoenix/src/phoenix/server/api/types/TimeSeries.py", line 152, in get_drift_timeseries_data
    return _get_timeseries_data(
  File "/Users/xandersong/phoenix/src/phoenix/server/api/types/TimeSeries.py", line 83, in _get_timeseries_data
    metric_instance.binning_method = binning.QuantileBinning(
  File "/Users/xandersong/phoenix/src/phoenix/metrics/binning.py", line 150, in __init__
    self.bins = self.numeric_bins(numeric_series)
  File "/Users/xandersong/phoenix/src/phoenix/metrics/binning.py", line 131, in numeric_bins
    quantiles = np.nanquantile(data, probabilities)
  File "<__array_function__ internals>", line 180, in nanquantile
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/nanfunctions.py", line 1541, in nanquantile
    return _nanquantile_unchecked(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/nanfunctions.py", line 1559, in _nanquantile_unchecked
    r, k = function_base._ureduce(a,
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 3725, in _ureduce
    r = func(a, **kwargs)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/nanfunctions.py", line 1581, in _nanquantile_ureduce_func
    result = _nanquantile_1d(part, q, overwrite_input, method)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/nanfunctions.py", line 1607, in _nanquantile_1d
    return function_base._quantile_unchecked(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 4424, in _quantile_unchecked
    r, k = _ureduce(a,
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 3725, in _ureduce
    r = func(a, **kwargs)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 4593, in _quantile_ureduce_func
    result = _quantile(arr,
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 4710, in _quantile
    result = _lerp(previous,
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/numpy/lib/function_base.py", line 4527, in _lerp
    diff_b_a = subtract(b, a)
TypeError: numpy boolean subtract, the `-` operator, is not supported, use the bitwise_xor, the `^` operator, or the logical_xor function instead.
Stack (most recent call last):
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/uvicorn/server.py", line 60, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/asyncio/base_events.py", line 1859, in _run_once
    handle._run()
  File "/Users/xandersong/.pyenv/versions/3.8.15/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/asgi/__init__.py", line 48, in __call__
    await self.http_handler_class(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/asgi/handlers/http_handler.py", line 50, in handle
    response = await self.get_http_response(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/asgi/handlers/http_handler.py", line 148, in get_http_response
    result = await execute(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/asgi/handlers/http_handler.py", line 201, in execute
    return await self.schema.execute(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/schema/schema.py", line 226, in execute
    result = await execute(
  File "/Users/xandersong/.virtualenvs/phoenixdev/lib/python3.8/site-packages/strawberry/schema/execute.py", line 144, in execute
    process_errors(result.errors, execution_context)
```

**Expected behavior**
Phoenix launches without error.

**Screenshots**
<img width="682" alt="Screenshot 2023-04-13 at 8 53 17 PM" src="https://user-images.githubusercontent.com/15664869/231937436-01c1648a-4ce2-4e97-be4f-b9b72014878a.png">

**Environment (please complete the following information):**
 - OS: MacOS
 - Notebook Runtime: Jupyter
 - Browser: Chrome

**Additional context**
I pushed a branch named `support-boolean-columns` that contains a notebook to repro.

**Comment by AmanSal1** [comment]
is this issue fixed ?

**Comment by mikeldking** [comment]
Hi @AmanSal1 thanks for checking in! We just triaged this bug and are working on the fix for the next release. Should be available no later than next week.

**Comment by mikeldking** [comment]
@AmanSal1 fixed in `v0.0.14`

```sh
pin install arize-phoenix==0.0.14
```
